azure_functions_worker/shared_memory_data_transfer/file_accessor_windows.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import mmap
+from typing import Optional
+from .file_accessor import FileAccessor
+from ..logging import logger
+
+
+class FileAccessorWindows(FileAccessor):
+    """
+    For accessing memory maps.
+    This implements the FileAccessor interface for Windows.
+    """
+    def open_mem_map(
+            self,
+            mem_map_name: str,
+            mem_map_size: int,
+            access: int = mmap.ACCESS_READ) -> Optional[mmap.mmap]:
+        """
+        Note: On Windows, an mmap is created if one does not exist even when
+              attempting to open it.
+        """
+        if mem_map_name is None or mem_map_name == '':
+            raise Exception(
+                f'Cannot open memory map. Invalid name {mem_map_name}')
+        if mem_map_size < 0:
```

Comment:
Is 0 valid?

Reply:
0 means open the entire map. Adding comment.